applications/TrilinosApplication/custom_python/add_trilinos_convergence_accelerators_to_python.cpp

```diff
@@ -0,0 +1,86 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//                   Philipp Bucher
+//
+//
+
+// System includes
+
+// External includes
+#include "Epetra_FEVector.h"
+
+// Project includes
+#include "solving_strategies/convergence_accelerators/convergence_accelerator.h"
+#include "spaces/ublas_space.h"
+#include "trilinos_space.h"
+
+// Application includes
+#include "trilinos_pointer_wrapper.h"
+#include "add_trilinos_convergence_accelerators_to_python.h"
+#include "custom_utilities/trilinos_mvqn_recursive_convergence_accelerator.hpp"
+#include "../FSIApplication/custom_utilities/aitken_convergence_accelerator.hpp" // To be removed
+
+namespace Kratos {
+namespace Python {
+
+typedef UblasSpace<double, Matrix, Vector> TrilinosLocalSpaceType;
+typedef TrilinosSpace<Epetra_FECrsMatrix, Epetra_FEVector> TrilinosSparseSpaceType;
+
+void AuxiliarUpdateSolution(
+    ConvergenceAccelerator<TrilinosSparseSpaceType, TrilinosLocalSpaceType> &dummy,
+    AuxiliaryVectorWrapper &rResidualVector,
+    AuxiliaryVectorWrapper &rIterationGuess)
+{
+    dummy.UpdateSolution(rResidualVector.GetReference(), rIterationGuess.GetReference());
+}
```

Comment:
Hm I am not sure why this is needed, but I would discuss it in a separate PR

Reply:
If I'm not wrong, we had some issues when passing the epetra vectors from python so we implemented this auxiliary `AuxiliaryVectorWrapper` handler. In any case this was implemented long time ago so we should probably check it again.

Reply:
ok lets discuss it separately then